extensions/iceberg/s3/src/main/java/io/deephaven/extensions/s3/CredentialsPropertyAdapterInternal.java

```diff
@@ -0,0 +1,29 @@
+//
```

Comment:
These adapters are interesting, in #5754 we are creating data instruction objects (`S3Instructions` etc.) from the properties maps. This is doing the inverse, right?